src/libraries/Microsoft.Extensions.FileProviders.Abstractions/ref/Microsoft.Extensions.FileProviders.Abstractions.cs

```diff
@@ -17,14 +17,14 @@ public partial interface IFileInfo
         System.DateTimeOffset LastModified { get; }
         long Length { get; }
         string Name { get; }
-        string PhysicalPath { get; }
+        string? PhysicalPath { get; }
         System.IO.Stream CreateReadStream();
     }
     public partial interface IFileProvider
     {
-        Microsoft.Extensions.FileProviders.IDirectoryContents GetDirectoryContents(string subpath);
+        Microsoft.Extensions.FileProviders.IDirectoryContents GetDirectoryContents(string? subpath);
         Microsoft.Extensions.FileProviders.IFileInfo GetFileInfo(string subpath);
-        Microsoft.Extensions.Primitives.IChangeToken Watch(string filter);
+        Microsoft.Extensions.Primitives.IChangeToken Watch(string? filter);
```

Comment:
as per https://github.com/dotnet/runtime/pull/57409 should these be non-nullable?

Reply:
Yes. Didn't want to change it everywhere until decision is finalized. I'll update this PR shortly.